pkg/knuu/instance_helper.go

```diff
@@ -80,11 +80,17 @@ func (i *Instance) Labels() map[string]string {
 }
 
 // deployService deploys the service for the instance
-func (i *Instance) deployService(ctx context.Context) error {
+func (i *Instance) deployService(ctx context.Context, portsTCP, portsUDP []int) error {
+	// a sidecar instance should use the parent instance's service
+	if i.isSidecar {
+		return ErrDeployingServiceForSidecar.WithParams(i.k8sName)
+	}
+
+	serviceName := i.k8sName
 	labels := i.getLabels()
-	selectorMap := i.getLabels()
+	labelSelectors := i.getLabels()
```

Comment:
since both are the same, we could use either one var or assign one to another. e.g. 

`labelSelectors := labels`